skaffold.yaml.template

```diff
@@ -0,0 +1,34 @@
+apiVersion: skaffold/v2alpha2
+kind: Config
+build:
+  tagPolicy:
+    envTemplate:
+      template: "{{.IMAGE_NAME}}:{{.IMAGE_TAG}}"
```

Comment:
I think there is a way to do this without needing to get the has into an env variable. Something like this:

```
tagPolicy:
    gitCommit: {}
```
From the gitCommit tagger: https://skaffold.dev/docs/pipeline-stages/taggers/

Reply:
I know, but i couldn't figure out how to change the tag in the helm chart, if you wanna take a stable at it that'd be great.